runtime/lua/vim/tshighlighter.lua

```diff
@@ -50,73 +52,62 @@ function TSHighlighter.new(query, bufnr, ft)
   return self
 end
 
+function TSHighlighter:get_hl_from_capture(capture)
+
+  local function is_highlight_name(capture_name)
```

Comment:
Can you move this function just out of this scope? No need to redefine it every time.

Reply:
Should be fixed.